[2021-07-08 12:09:22,956] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:09:15.126213+00:00 [queued]>
[2021-07-08 12:09:22,977] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:09:15.126213+00:00 [queued]>
[2021-07-08 12:09:22,977] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 12:09:22,978] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-07-08 12:09:22,978] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-08 12:09:22,987] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): dump_user_log> on 2021-07-08T12:09:15.126213+00:00
[2021-07-08 12:09:22,992] {standard_task_runner.py:52} INFO - Started process 13598 to run task
[2021-07-08 12:09:23,000] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bluepi_dag', 'dump_user_log', '2021-07-08T12:09:15.126213+00:00', '--job-id', '80', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bluepi_dag.py', '--cfg-path', '/tmp/tmpb8udy63j', '--error-file', '/tmp/tmph7n5hprj']
[2021-07-08 12:09:23,003] {standard_task_runner.py:77} INFO - Job 80: Subtask dump_user_log
[2021-07-08 12:09:23,072] {logging_mixin.py:104} INFO - Running <TaskInstance: bluepi_dag.dump_user_log 2021-07-08T12:09:15.126213+00:00 [running]> on host 3cf0296804bd
[2021-07-08 12:09:23,139] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=natchanon
AIRFLOW_CTX_DAG_ID=bluepi_dag
AIRFLOW_CTX_TASK_ID=dump_user_log
AIRFLOW_CTX_EXECUTION_DATE=2021-07-08T12:09:15.126213+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-08T12:09:15.126213+00:00
[2021-07-08 12:09:23,151] {base.py:78} INFO - Using connection to: id: my_postgres_conn. Host: 35.247.174.171, Port: 5432, Schema: postgres, Login: exam, Password: ***, extra: {}
[2021-07-08 12:09:24,883] {python.py:151} INFO - Done. Returned value was: None
[2021-07-08 12:09:24,896] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bluepi_dag, task_id=dump_user_log, execution_date=20210708T120915, start_date=20210708T120922, end_date=20210708T120924
[2021-07-08 12:09:24,933] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-08 12:09:24,945] {local_task_job.py:151} INFO - Task exited with return code 0
